src/test/rustdoc/fn-bound.rs

```diff
@@ -0,0 +1,19 @@
+use std::iter::Peekable;
```

Comment:
Nitpicking: can you add `Regression test for #100143` in a comment please?

Reply:
Done

Reply:
Thanks!